entries/p/pdffiller.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "pdfFiller": {
+    "domain": "pdffiller.com",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://blog.pdffiller.com/two-factor-authentication-phone-login/",
```

Comment:
From reading the article, it sounds more like they offer SMS authentication as an alternative to password authentication. Can you advise if this is indeed the case or if SMS codes are required in addition to a password?

Reply:
I tried it and found that my phone number could not be registered. I think we could put this PR in the issue because I currently have no way of receiving the code sent to my phone. 😢

Reply:
If we can't verify that they really offer 2FA, I think it would be best if we close this PR for now.

Reply:
It's okay! I'll check again later and contact you if I have any news! ❤️